src/exposurelog/testutils.py

```diff
@@ -391,6 +392,23 @@ def random_date(precision: int = 0) -> datetime.datetime:
     ).datetime
 
 
+def random_obs_id() -> str:
+    """Return a random obs_id.
+
+    The format is aa_a_dddddddd_dddddd, where:
+
+    * a is an uppercase letter
```

Comment:
Perhaps it would be clearer to describe the format as `AA_A_dddddddd_dddddd` so from looking at that format string it already is suggested that A is an uppercase letter? At first I got the impression that `aa_` are lower case and only `_a_` is uppercase.